src/commcare_cloud/commcare_cloud.py

```diff
@@ -175,19 +175,24 @@ def call_commcare_cloud(input_argv=sys.argv):
     force_python_2 = "--force-commcare-cloud-to-use-python2"
     if not os.environ.get("TRAVIS_TEST") and sys.version_info[0] == 2:
         if force_python_2 not in input_argv:
-            from textwrap import dedent
-            print(dedent("""
+            exit(dedent("""
                 Error: you must upgrade to Python 3. Though not desirable, if you really have
```

Comment:
I think we should also update this error output to reflect the official drop date.